mlrun/artifacts/base.py

```diff
@@ -379,6 +379,7 @@ def get_store_url(self, with_tag=True, project=None, with_tree=True):
             iter=self.metadata.iter,
             tree=tree,
             tag=tag,
+            uid=self.metadata.uid,
```

Comment:
Why not to use your new property
```suggestion
            uid=self.uid,
```